packages/web/src/components/createCell.tsx

```diff
@@ -71,27 +71,17 @@ export type CellFailureProps<TVariables extends OperationVariables = any> = {
   updating?: boolean
 }
 
-// aka guarantee that all properties in T exist
-// This is necessary for Cells, because if it doesn't exist it'll go to Empty or Failure
-type Guaranteed<T> = {
-  [K in keyof T]-?: NonNullable<T[K]>
-}
-
 /**
  * Use this type, if you are forwarding on the data from your Cell's Success component
- * Because Cells automatically checks for "empty", or "errors" - if you receive the data type in your
- * Success component, it means the data is guaranteed (and non-optional)
```

Comment:
These comments are no longer accurate because we render `Success` if _any_ property is present/non-empty.  Other properties can still be missing/empty.

A custom `isEmpty` implementation or a missing `Empty` definition can also result in `Success`.